assets/airports/lszh.json

```diff
@@ -0,0 +1,1812 @@
+            "name": "Negra One Alpha",
+            "entryPoints": {
+                "NEGRA": ["NEGRA"],
+                "RAVED": ["RAVED", "ETOXU", "NEGRA"]
+            },
+            "body": ["MATIV", ["@AMIKI", "A70+"]],
+            "rwy": {
+                "LSZH10": [],
+                "LSZH14": [],
+                "LSZH16": [],
+                "LSZH28": [],
+                "LSZH32": [],
+                "LSZH34": []
+            },
+            "draw": [
+                ["RAVED", "ETOXU", "NEGRA", "MATIV", "AMIKI"]
+            ]
+        },
+        "RILA1A": {
+            "icao": "RILA1A",
+            "name": "Rilax One Alpha",
+            "entryPoints": {
+                "RILAX": ["RILAX"]
+            },
+            "body": ["LAMAX", ["@AMIKI", "A70+"]],
+            "rwy": {
+                "LSZH10": [],
+                "LSZH14": [],
+                "LSZH16": [],
+                "LSZH28": [],
+                "LSZH32": [],
+                "LSZH34": []
+            },
+            "draw": [
+                ["RILAX", "LAMAX", "AMIKI"]
+            ]
+        },
+        "TRA1Z": {
+            "icao": "TRA1Z",
+            "name": "Trasadingen One Zulu",
+            "entryPoints": {
+                "TRA": ["TRA"]
+            },
+            "body": [["@AMIKI", "A70+"]],
+            "rwy": {
+                "LSZH10": [],
+                "LSZH14": [],
+                "LSZH16": [],
+                "LSZH28": [],
+                "LSZH32": [],
+                "LSZH34": []
+            },
+            "draw": [
+                ["TRA", "AMIKI"]
+            ]
+        },
+        "URNA2Z": {
+            "icao": "URNA2Z",
+            "name": "Urnas Two Zulu",
+            "entryPoints": {
+                "URNAS": [["URNAS", "A140+"]]
+            },
+            "body": [["ALBIX", "A100+"], ["ZH650", "A80+"], "ZH676", ["@GIPOL", "A70+"]],
+            "rwy": {
+                "LSZH10": [],
+                "LSZH14": [],
+                "LSZH16": [],
+                "LSZH28": [],
+                "LSZH32": [],
+                "LSZH34": []
+            },
+            "draw": [
+                ["URNAS", "ALBIX", "ZH650", "ZH676", "GIPOL"]
+            ]
+        },
+        "WIL2Z": {
+            "icao": "WIL2Z",
+            "name": "Willisau Two Zulu",
+            "entryPoints": {
+                "WIL": ["WIL"]
+            },
+            "body": [["@GIPOL", "A70+"]],
+            "rwy": {
+                "LSZH10": [],
+                "LSZH14": [],
+                "LSZH16": [],
+                "LSZH28": [],
+                "LSZH32": [],
+                "LSZH34": []
+            },
+            "draw": [
+                ["WIL", "GIPOL"]
+            ]
+        }
+    },
+    "spawnPatterns": [
+        {
+            "origin": "LSZH",
+            "destination": "",
+            "category": "departure",
+            "route": "LSZH.DEGES2W.DEGES..ROMGA..BODAN..ALAGO..BIKBI..MINGA..SUDEN",

```

Comment:
The runway should be included in the entry here, so `LSZH28.DEGES2W.DEGES`, just like it is shown in the procedure defintion you created ealier in the file.